sapi/fpm/fpm/fpm_conf.c

```diff
@@ -213,8 +213,21 @@ static int fpm_conf_expand_pool_name(char **value) {
 static char *fpm_conf_set_boolean(zval *value, void **config, intptr_t offset) /* {{{ */
 {
 	zend_string *val = Z_STR_P(value);
-	bool value_y = zend_string_equals_literal(val, "1");
-	bool value_n = ZSTR_LEN(val) == 0; /* Empty string is the only valid false value */
+	/* we need to check all alowed values to correctly set value from the environment varaible */
+	bool value_y = (
+		zend_string_equals_literal(val, "1") ||
+		zend_string_equals_literal(val, "yes") ||
+		zend_string_equals_literal(val, "true") ||
+		zend_string_equals_literal(val, "on")
+	);
+	bool value_n = (
+		value_y || ZSTR_LEN(val) == 0 ||
+		zend_string_equals_literal(val, "no") ||
```

Comment:
If you accept 1 for a truthy value, why not 0 for a falsy value?

Reply:
I didn't add it because it's not what INI considers as true but you are right that it makes more sense so added that in committed version.